### PR TITLE
Changes resulting from OData-Aggr walkthrough

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

File: examples/Org.OData.Aggregation.V1.SalesModel-sample.xml [hunk 1]
```diff
@@ -81,13 +93,20 @@
...
+          </Annotation>
+        </Property>
+        <NavigationProperty Name="Currency" Type="SalesModel.Currency" Nullable="false" />
         <NavigationProperty Name="SalesOrganization" Type="SalesModel.SalesOrganization" Partner="Sales" />
         <NavigationProperty Name="Product" Type="SalesModel.Product" Partner="Sales" />
         <NavigationProperty Name="Customer" Type="SalesModel.Customer" Partner="Sales" />
         <NavigationProperty Name="Time" Type="SalesModel.Time" />
       </EntityType>
-      <EntityContainer Name="Container">
+      <EntityContainer Name="SalesModel">
```
Comment (ralfhandl) [inline examples/Org.OData.Aggregation.V1.SalesModel-sample.xml:109]: Same name as the containing schema, which might cause confusion, for example in 
~~~xml
<Annotation Target="SalesModel.SalesModel/Sales"
~~~
Why not leave it as "Container" or "defaultContainer"?

Comment (ralfhandl) [reply examples/Org.OData.Aggregation.V1.SalesModel-sample.xml:109]: Also now the annotation target in line 131 is broken, still says `Target="SalesModel.Container/Sales"`

**Review by ralfhandl — CHANGES_REQUESTED** [review]

File: examples/Org.OData.Aggregation.V1.SalesModel-sample.xml (hunk 1 above)
Comment (HeikoTheissen) [reply examples/Org.OData.Aggregation.V1.SalesModel-sample.xml:109]: > Why not leave it as "Container" or "defaultContainer"?

I wanted to follow section 6.4 in OData-Aggr.

**Review by ralfhandl — APPROVED** [review]

**Review by ralfhandl — APPROVED** [review]

**Review by ralfhandl — APPROVED** [review]

**Comment by ralfhandl** [comment]
@HeikoTheissen Objections to extracting a TypeDefinition `Transformation` to be used in property `ApplySupportedBase/Transformations`?

Then we would see the list of allowed values in the Markdown?

File: vocabularies/Org.OData.Aggregation.V1.xml [hunk 2]
```diff
@@ -311,19 +343,25 @@
       </Term>
 
       <Term Name="LeveledHierarchy" Type="Collection(Edm.PropertyPath)" Nullable="false" AppliesTo="EntityType ComplexType">
-        <Annotation Term="Core.Description" String="Defines a leveled hierarchy [OData-Aggr, section 5.5.1]" />
+        <Annotation Term="Core.Description" String="Defines a leveled hierarchy [OData-Data-Agg-v4.0, section 5.5.1]" />
```
Comment (ralfhandl) [inline vocabularies/Org.OData.Aggregation.V1.xml:346]: Should this `[...]` reference be a hyperlink?

File: vocabularies/Org.OData.Aggregation.V1.xml [hunk 3]
```diff
@@ -311,19 +343,25 @@
       </Term>
 
       <Term Name="LeveledHierarchy" Type="Collection(Edm.PropertyPath)" Nullable="false" AppliesTo="EntityType ComplexType">
-        <Annotation Term="Core.Description" String="Defines a leveled hierarchy [OData-Aggr, section 5.5.1]" />
+        <Annotation Term="Core.Description" String="Defines a leveled hierarchy [OData-Data-Agg-v4.0, section 5.5.1]" />
       </Term>
 
       <Term Name="RecursiveHierarchy" Type="Aggregation.RecursiveHierarchyType" Nullable="false" AppliesTo="EntityType">
-        <Annotation Term="Core.Description" String="Defines a recursive hierarchy [OData-Aggr, section 5.5.2]" />
+        <Annotation Term="Core.Description" String="Defines a recursive hierarchy [OData-Data-Agg-v4.0, section 5.5.2]" />
```
Comment (ralfhandl) [inline vocabularies/Org.OData.Aggregation.V1.xml:350]: Should this `[...]` reference be a hyperlink?

**Review by ralfhandl — CHANGES_REQUESTED** [review]

File: vocabularies/Org.OData.Aggregation.V1.xml (hunk 3 above)
Comment (HeikoTheissen) [reply vocabularies/Org.OData.Aggregation.V1.xml]: No.

**Review by ralfhandl — APPROVED** [review]

**Pull request merged**